lib/api/bucketDeleteCors.js

```diff
@@ -44,18 +47,18 @@ export default function bucketDeleteCors(authInfo, request, log, callback) {
             });
             // TODO: add 'deleteBucketCors' to Utapi Client action map
             // pushMetric('deleteBucketCors', log, { bucket: bucketName });
-            return callback();
+            return callback(null, corsHeaders);
         }
 
         log.trace('deleting cors configuration in metadata');
         bucket.setCors(null);
         return metadata.updateBucket(bucketName, bucket, log, err => {
             if (err) {
-                return callback(err);
+                return callback(err, corsHeaders);
```

Comment:
is this what aws does?  sort of odd that you return the cors headers based on the configuration you are deleting